internal/sbi/api_datarepository.go

```diff
@@ -0,0 +1,2577 @@
+			strings.ToUpper("Get"),
+			"/exposure-data/:ueId/access-and-mobility-data",
+			s.HandleQueryAccessAndMobilityData,
+		},
+
+		{
+			"ExposureDataSubsToNotifyPost",
+			strings.ToUpper("Post"),
+			"/exposure-data/subs-to-notify",
+			s.HandleExposureDataSubsToNotifyPost,
+		},
+
+		{
+			"ExposureDataSubsToNotifySubIdDelete",
+			strings.ToUpper("Delete"),
+			"/exposure-data/subs-to-notify/:subId",
+			s.HandleExposureDataSubsToNotifySubIdDelete,
+		},
+
+		{
+			"ExposureDataSubsToNotifySubIdPut",
+			strings.ToUpper("Put"),
+			"/exposure-data/subs-to-notify/:subId",
+			s.HandleExposureDataSubsToNotifySubIdPut,
+		},
+
+		/* appRoutes */
+		{
+			"ApplicationDataInfluenceDataSubsToNotifyGet",
+			strings.ToUpper("Get"),
+			"/application-data/influenceData/subs-to-notify",
+			s.HandleApplicationDataInfluenceDataSubsToNotifyGet,
+		},
+
+		{
+			"ApplicationDataInfluenceDataSubsToNotifyPost",
+			strings.ToUpper("Post"),
+			"/application-data/influenceData/subs-to-notify",
+			s.HandleApplicationDataInfluenceDataSubsToNotifyPost,
+		},
+
+		{
+			"ApplicationDataInfluenceDataInfluenceIdDelete",
+			strings.ToUpper("Delete"),
+			"/application-data/influenceData/:influenceId",
+			s.HandleApplicationDataInfluenceDataInfluenceIdDelete,
+		},
+
+		{
+			"ApplicationDataInfluenceDataInfluenceIdPatch",
+			strings.ToUpper("Patch"),
+			"/application-data/influenceData/:influenceId",
+			s.HandleApplicationDataInfluenceDataInfluenceIdPatch,
+		},
+
+		{
+			"ApplicationDataInfluenceDataInfluenceIdPut",
+			strings.ToUpper("Put"),
+			"/application-data/influenceData/:influenceId",
+			s.HandleApplicationDataInfluenceDataInfluenceIdPut,
+		},
+
+		{
+			"ApplicationDataInfluenceDataInfluenceIdPut",
+			strings.ToUpper("Post"),
+			"/application-data/influenceData/:influenceId",
+			s.HandleApplicationDataInfluenceDataInfluenceIdPost,
+		},
+	}
+}
+
+// Index is the index handler.
+func Index(c *gin.Context) {
+	c.String(http.StatusOK, "Hello World!")
+}
+
+// HTTPAmfContext3gpp - To modify the AMF context data of a UE using 3gpp access in the UDR
+func (s *Server) HandleAmfContext3gpp(c *gin.Context) {
+	var patchItemArray []models.PatchItem
+
+	requestBody, err := c.GetRawData()
+	if err != nil {
+		problemDetail := models.ProblemDetails{
+			Title:  "System failure",
+			Status: http.StatusInternalServerError,
+			Detail: err.Error(),
+			Cause:  "SYSTEM_FAILURE",
+		}
+		logger.DataRepoLog.Errorf("Get Request Body error: %+v", err)
+		c.JSON(http.StatusInternalServerError, problemDetail)
+		return
+	}
+
+	err = openapi.Deserialize(&patchItemArray, requestBody, "application/json")
+	if err != nil {
+		problemDetail := "[Request Body] " + err.Error()
+		rsp := models.ProblemDetails{
+			Title:  "Malformed request syntax",
+			Status: http.StatusBadRequest,
+			Detail: problemDetail,
+		}
+		logger.DataRepoLog.Errorln(problemDetail)
+		c.JSON(http.StatusBadRequest, rsp)
+		return
+	}
+
+	logger.DataRepoLog.Infof("Handle AmfContext3gpp")
```

Comment:
Log level should be trace.
```suggestion
	logger.DataRepoLog.Tracef("Handle AmfContext3gpp")
```